### PR TITLE
picshow.py: Fix a bug

### LINKED ISSUE #179: Fix display pics command
If I ask Jarvis to display pics of "Sachin Ramesh", It should show [this](https://www.google.co.in/search?tbm=isch&sa=1&q=Sachin+Ramesh&oq=Sachin+Ramesh&gs_l=psy-ab.3..0i13k1l4.1083.2192.0.2392.7.7.0.0.0.0.335.662.0j2j0j1.3.0....0...1.1.64.psy-ab..5.2.482...0i7i30k1j0i7i10i30k1.OYUVlolj_d8) instead of [this](https://www.google.co.in/search?tbm=isch&sa=1&q=Ramesh&oq=Ramesh&gs_l=psy-ab.3..0l4.135990.135990.0.136421.1.1.0.0.0.0.136.136.0j1.1.0....0...1.1.64.psy-ab..0.1.135.1SS0cC9xJsI). 

One way to fix this:
after entering "show pics" command.
Prompt for the query he/she need to search for.

### PR DESCRIPTION
**Pull request by appi147**
Fix a bug which was showing only last word of query in search result
Closes and fixes #179

### CODE REVIEW

**Comment by appi147** [comment]
@sukeesh Please try one time by running. It works

File: jarviscli/packages/picshow.py [hunk 1]
```diff
@@ -3,7 +3,5 @@
 
 
 def showpics(string):
-    string = string.split(" ")
-    query = string[-1]
```
Comment (appi147) [inline jarviscli/packages/picshow.py]: Earlier this caused last element of list `query`

File: jarviscli/packages/picshow.py [hunk 2]
```diff
@@ -3,7 +3,5 @@
...
-    url = "https://www.google.com/search?tbm=isch&q={}".format(query)
+    url = "https://www.google.com/search?tbm=isch&q={}".format(string.replace("of",""))
```
Comment (appi147) [inline jarviscli/packages/picshow.py]: This searches for whole strings and removing of from beginning

**Comment by appi147** [comment]
@sukeesh Is it working now? Shall I merge?

**Comment by sukeesh** [comment]
@appi147 It says 
screen shot 2017-07-30 at 9 12 12 pm

**Comment by appi147** [comment]
use display instead of show

_Sent from my Xiaomi 2014818 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**